### PR TITLE
Feat: add test for ipt pallet

### PR DESCRIPTION
**Pull request by SHA888**
- Add tests.rs in ipt pallet
- Add mock.rs in ipt pallet

### CODE REVIEW

**Review by Invarchitect — APPROVED** [review]
Awesome work, looks good. Thank you very much, Kresna!

File: ipt/src/lib.rs [hunk 1]
```diff
@@ -27,6 +27,9 @@ use sp_runtime::{
 };
 use sp_std::{convert::TryInto, vec::Vec};
 
+mod mock;
```
Comment (arrudagates) [inline ipt/src/lib.rs:30]: I think it's better if you put `#[cfg(test)]` here and in the line below instead of in the respective files

**Review by arrudagates — CHANGES_REQUESTED** [review]
These tests are great!

File: ipt/src/lib.rs (hunk 1 above)
Comment (SHA888) [reply ipt/src/lib.rs:31]: Thanks @arrudagates I've made the changes

File: ipt/src/mock.rs [hunk 2]
```diff
@@ -0,0 +1,98 @@
+//! Mocks for the gradually-update module.
+
+#![cfg(test)]
```
Comment (arrudagates) [inline ipt/src/mock.rs:3]: You can remove these now, the whole file will skip compilation when tests arent running

File: ipt/src/tests.rs [hunk 3]
```diff
@@ -0,0 +1,118 @@
+//! Unit tests for the IPT pallet.
+
+#![cfg(test)]
```
Comment (arrudagates) [inline ipt/src/tests.rs:3]: You can remove these now, the whole file will skip compilation when tests arent running

**Review by arrudagates — APPROVED** [review]
Approved for merge!

**Pull request merged**